vermouth/tests/test_graph_utils.py

```diff
@@ -639,10 +632,14 @@ def test_blockmodel_graph_attr():
     """
     graph = basic_molecule([{}, {}, {}], {(0, 1): {}, (1, 2): {}})
     found = vermouth.graph_utils.blockmodel(graph, [[0], [1, 2]])
-    assert 1 == found.nodes[0]['nnodes'] == len(found.nodes[0]['graph'].nodes)
-    assert 2 == found.nodes[1]['nnodes'] == len(found.nodes[1]['graph'].nodes)
-    assert 0 == found.nodes[0]['nedges'] == len(found.nodes[0]['graph'].edges)
-    assert 1 == found.nodes[1]['nedges'] == len(found.nodes[1]['graph'].edges)
+    assert 1 == found.nodes[0]['nnodes']
```

Comment:
I was more thinking about how it was before:

```
assert found.nodes[0]['nnodes'] == len(found.nodes[0]['graph'].nodes) == 1
```

Then, in case of failure, pytest says that `found.nodes[0]['nnodes']` does not have the expected value instead of saying that `1` is at fault.

A minor detail, though.